src/libm/rename_llvm.h

```diff
@@ -0,0 +1,149 @@
+//          Copyright Naoki Shibata 2010 - 2017.
+// Distributed under the Boost Software License, Version 1.0.
+//    (See accompanying file LICENSE.txt or copy at
+//          http://www.boost.org/LICENSE_1_0.txt)
+
+#define xsin __llvm_sin_f64
```

Comment:
Yes - it is better not to introduce code before we have agreed on the interface.